upup/pkg/fi/cloudup/openstack/instance.go

```diff
@@ -65,7 +65,7 @@ func (c *openstackCloud) CreateInstance(opt servers.CreateOptsBuilder) (*servers
 	}
 }
 
-func (c *openstackCloud) ListServerFloatingIPs(instanceID string) ([]*string, error) {
+func (c *openstackCloud) ListServerIPs(instanceID string, IPType string) ([]*string, error) {
```

Comment:
I mean I like the generalization of that method, but there is no call using it without `FloatingType` in this PR.

Reply:
yeah my plan was to delete port also when deleting instance, but it was not that easy - so basically that is ready there now for next try :)